src/commands/rover/RestartCommand.js

```diff
@@ -30,9 +31,8 @@ class RestartCommand extends Command {
       await msg.reply('Restarting...')
       process.exit()
     }
-    const shard = msg.client.shard.shardIDForGuildID(msg.guild.id, msg.client.shard.count).catch(() => {
-      msg.reply('An invalid server id was given!')
-    })
+    if(isNaN(args.server)) return msg.reply("An invalid server id was given!")
+    const shard = Discord.ShardClientUtil.shardIDForGuildID(args.server, msg.client.shard.count)
```

Comment:
```suggestion
    const shard = ShardClientUtil.shardIDForGuildID(args.server, msg.client.shard.count)
```

I forgot this part 🤦‍♀️